content/04.results.md

```diff
@@ -13,15 +13,39 @@ Stub for the process for contributing analytical code (*Figure {@fig:FigS1}S1* a
 
 ![Overview of the OpenPBTA Project.](images/main/Figure1.png){#fig:Fig1}
 
+### Somatic Mutational Landscape of Pediatric Brain Tumors
 
-### Landscape of Genomic Alterations
+We  performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.
 
-The OncoPrint will provide a visualization of the genomic alterations found in the analyses implemented throughout the OpenPBTA project.
+#### Low-grade astrocytic tumors
+
+*Figure {@fig:Fig2}2A* depicts an oncoprint of driver genes for 227 primary low-grade astrocytic tumors. As expected, the majority (62%, X/227) of these tumors harbor a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver. 
+We observed additional mutations in _FGFR1_ (2%), _PIK3CA_ (2%), _KRAS_ (2%), _TP53_ (1%), and _ATRX_ (1%) and fusions in _NTRK2_ (2%), _RAF1_ (2%), _MYB_ (1%), _QKI_ (1%), _ROS1_ (1%), and _FGFR2_ (1%), concordant with previous studies highlighting the near universal upregulation of the RAS/MAPK pathway in these tumors [@doi:10.1186/s40478-020-00902-z; @doi:10.1016/j.ccell.2020.03.011]. 
+Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signaling pathway in LGATs (*Figure {@fig:Fig4}4B).
+
+#### Embryonal tumors (N = 131)
+*Figure {@fig:Fig2}2B* depicts the oncoprint for 131 primary embryonal tumors within the OpenPBTA. 
+The majority (N = 97) are medulloblastomas and span the spectrum of molecular subtypes: WNT, SHH, Group3, and Group 4 (see *Molecular Subtyping of CNS Tumors*) as well as mutational landscape. 
+We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
+
+#### Diffuse astrocytic and oligodendroglial tumors
+In *Figure {@fig:Fig2}2C*, we show an oncoprint for diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 29) biopsied at diagnosis. The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
```

Comment:
I don't feel strongly about this, but all the other sections start the same way, so this one stood out. 
```suggestion
*Figure {@fig:Fig2}C* depicts an oncoprint for diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 29) biopsied at diagnosis.
The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
```